mempool/clist_mempool.go

```diff
@@ -397,11 +405,11 @@ func (mem *CListMempool) resCbFirstTime(
 			}
 
 			// Check transaction not already in the mempool
-			if e, ok := mem.txsMap.Load(types.Tx(tx).Key()); ok {
-				memTx := e.(*clist.CElement).Value.(*mempoolTx)
+			if elem, ok := mem.txsMap.Load(types.Tx(tx).Key()); ok {
```

Comment:
Any real reason for renaming this variable?

Reply:
This is also to keep the naming consistent. The mempool holds `CElement`s and in other places we use the name `elem` for functions returning `CElement`s. But we can keep it as it is if you prefer.

Reply:
Reverted.